cloud/pkg/devicecontroller/controller/downstream.go

```diff
@@ -167,6 +175,119 @@ func (dc *DownstreamController) syncDevice() {
 	}
 }
 
+// addDeviceMapper is function to add deviceMapper deployment
+func (dc *DownstreamController) addDeviceMapper(device *v1alpha2.Device, nodename string) {
+	protocol, err := getProtocolNameOfDevice(device)
```

Comment:
add a pre check to check whether the device has image annotation just like below?
```
annotations:
    image: 'kubeedge/kubeedge-pi-counter:v1.0.0'
```
If not, we can just return and do nothing, no need to continue the next processing.